### DISCUSSION TITLE
Presenting Jamulus Concept & Requirements to Telecom Operators?

### DISCUSSION THREAD

**Discussion by RoSchu500** [jamulussoftware/jamulus#1597] [Technical Support & Help]
Hi there, I have just started using Jamulus for my Jazz-Band-Rehearsals and I am really exited about it.
I am looking for somebody who could present Jamulus at a telecom operator workshop. I would like the telecom operators to understand the needs of the Jamulus community and improve their service quality accordingly. 

Is there anybody in the community who could present the Jamulus concepts and requirements? 
The workshop will take place online on 18 and 19 May (11am - 2pm EDT / 17:00 - 20:00 CEST). A 15 minute presentation in English would be fine.

As there are quite a few global telecom providers represented at the workshop, it could make a difference for the community going forward. In case you do not want to present yourself, please do not hesitate to point to appropriate community members I could approach.

Many thanks for your feedback and suppor!
Rolf.

**Comment by ghost** [comment]
It would be nice to have a new Jamulus release for the presentor at this event.

**Comment by gilgongo** [comment]
Thanks for this, Rolf. What sort of things would the telecom operators be interested in? Is it the more technical side (the protocol and the technical architecture), or the social side (the use of public/private servers for bands and chors, WordJam, etc.)?

**Reply by RoSchu500** [reply]
Thanks for your message. It would be great to convey three messages: 
- community success of Jamulus (number of servers, users etc.) --> to illustrate the market relevance
- high level technical architecture --> it is important to make clear that Jamulus works with central servers that connects to all musicians (the workshop is about edge computing)
- current situation with telecom services (latency, jitter, throughput, interconnect etc.) and requirements from Jamulus for the telecom service providers (perhaps including wishlist for 5G)
Does that help. Would you be interested to provide the presentation? Or do you know somebody who could? If so we have to be quick and perhaps set-up a introduction web-session.
What do you think?

**Reply by hoffie** [reply]
Sounds like you are already covering several of the important points so maybe you would be all set for giving this presentation yourself? There's also a paper on the website. There should also be architectural diagrams floating around. :)

What kinds of operators will be participating there? Are they regional or rather country-wide or even global operators? What kind of position do the participants have?

**Reply by gene96817** [reply]
I'd be interested in seeing the finished presentation. Maybe it could be helpful with my local ISPs.

**Reply by ghost** [reply]
Hi, we don't know anyone, or even where or or which Telecom workshop this is.
I'm starting to think (given the comments and time available) that Nothing is going to happen. Please, @RoSchu500, give details and get the ball rolling. BTW is @corrados aware of this?

**Reply by hoffie** [reply]
> BTW is corrados aware of this?

corrados is currently no longer active in the project, see https://github.com/jamulussoftware/jamulus/discussions/928

**Reply by RoSchu500** [reply]
Thanks for all the comments. We have confirmed participants from Verizon, British Telekom, Vodafone, Deutsche Telekom and from smaller players like Proximus. We prefer an external speaker that can cover the practical experiences with Jamulus including the challenges on the network side. Yes, this is getting tight now. We will close the agenda by Monday (10 May). I look forward to hear from you. No problem if that does not work out this time, we run this workshop twice a year...

**Reply by gene96817** [reply]
what is the name of the conference/workshop?

**Reply by ghost** [reply]
The person that chooses to do the presentation shall declare that they are doing the presentation before may 10. Then it is expected that RoSchu500 will come through with the rest of the bargain. Otherwise, this will be a flash in the pan.

**Reply by RoSchu500** [reply]
The somewhat unspecific title of the workshop is Open Edge Computing Spring Workshop 2021. This refers to the organiser   www.openedgecomputing.org, an industry consortium that shapes and drives the success of edge computing.
I guess it is too late to add a speaker on Jamulus to the agenda now. Still, please do let me know if you are interested to talk. Our next workshop will take place in December 21. Thanks for your feedback and interest...

**Reply by gene96817** [reply]
I am very interested in following your efforts to deliver a Jamulus presentation.

My concern is that the service providers care more about edge computing that will advance their revenue growth. Regardless, it would be good to create visibility for the Jamulus requirements and the Jamulus community. Out here in my area, I am working on networking social requirements (which also requires good edge computing). This discussion is built on the recognition that the pandemic has exposed that the Internet service quality is not well suited to support our social requirements (the popular jargon out here is "livelihood").  For Zoom, Webex, and Jamulus, this is about low jitter, symmetric traffic. (Note, if the networks deliver this, we would also get good ping times.)

**Comment by pcar75** [comment]
May I suggest talking to the Melomax people ("Jamulus-as-a-Service").  They have probably presented and refined a few slideshows for exactly this.  I understand that they have (or had) close ties with Jamulus developers and their advice wouldn't hurt.

**Reply by alexbuckland** [reply]
Hello, the melomax people here 👋🏻
It seems we've missed out on presenting this time round @RoSchu500, but lets talk about these three points you mention and if the community has no strong objections I would be happy to present next time in December

> 1. community success of Jamulus (number of servers, users etc.) --> to illustrate the market relevance
> 1. high level technical architecture --> it is important to make clear that Jamulus works with central servers that connects to all musicians (the workshop is about edge computing) 
> 1. current situation with telecom services (latency, jitter, throughput, interconnect etc.) and requirements from Jamulus for the telecom service providers (perhaps including wishlist for 5G)

1. Should be able to get this from https://explorer.jamulus.io for public servers. Are there any peak numbers around for this?
2. [Architecture diagram](https://jamulus.io/wiki/Getting-Started#how-jamulus-works-in-general) and [bandwidth diagram](https://jamulus.io/wiki/Network-Requirements#network-bandwidth) 
3. I would need some help gathering data for this from the community.

**Reply by gilgongo** [reply]
On 3, I don't know whether it's worth mentioning that the project had to introduce a way of compensating for the fact that one ISP in the UK was causing packets to arrive out of order. This in turn caused audio issues for those on that ISP - see https://github.com/jamulussoftware/jamulus/issues/619

**Reply by dcorson-ticino-com** [reply]
> On 3, I don't know whether it's worth mentioning that the project had to introduce a way of compensating for the fact that one ISP in the UK was causing packets to arrive out of order. This in turn caused audio issues for those on that ISP - see #619

I would rather say that one provider was particularly bad, but this packet reordering helped in many cases and I now almost never hear that characteristic "underwater" sound of out-of-order packets any more.
